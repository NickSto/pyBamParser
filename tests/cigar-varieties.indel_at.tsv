chrM	200	D
chrM	202	I
chrM	211	ID
chrM	212	D
chrM	1901	D
chrM	6111	D
chrM	6809	D
chrM	7606	D
chrM	13605	I
chrM	14885	D
