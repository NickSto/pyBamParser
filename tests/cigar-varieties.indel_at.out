M01368:8:000000000-A3GHV:1:1103:15072:19679	1	251M
200	D	False
202	I	False
211	ID	False
212	D	False
M01368:8:000000000-A3GHV:1:1101:6679:24488	1	3S248M
200	D	False
202	I	False
211	ID	False
212	D	False
M01368:8:000000000-A3GHV:1:1101:20741:16339	1	199M1D2M2I8M2D2M38S
200	D	True
202	I	True
211	ID	True
212	D	True
M01368:8:000000000-A3GHV:1:1112:20014:11702	1	52S199M
200	D	False
M01368:8:000000000-A3GHV:1:2111:22092:13076	1	10S156M85S
M01368:8:000000000-A3GHV:1:2105:12460:13136	2	4M2I245M
200	D	False
202	I	False
211	ID	False
212	D	False
M01368:8:000000000-A3GHV:1:1112:22817:23465	31	251M
200	D	False
202	I	False
211	ID	False
212	D	False
M01368:8:000000000-A3GHV:1:2108:17157:13067	111	205M46S
200	D	False
202	I	False
211	ID	False
212	D	False
M01368:8:000000000-A3GHV:1:1104:17482:12333	199	112M1I138M
200	D	False
202	I	False
211	ID	False
212	D	False
M01368:8:000000000-A3GHV:1:1101:12899:15963	554	38S3M3I207M
M01368:8:000000000-A3GHV:1:1107:12635:16957	1785	116M1D135M
1901	D	True
M01368:8:000000000-A3GHV:1:1101:10617:26546	2526	11S3M1D237M
M01368:8:000000000-A3GHV:1:2107:13849:20881	2883	224M1D26M1S
M01368:8:000000000-A3GHV:1:1101:10077:7324	2941	166M1D85M
M01368:8:000000000-A3GHV:1:1104:16663:10687	4099	171M1I11M68S
M01368:8:000000000-A3GHV:1:1112:15168:23819	5975	10S5M1I1M1I233M
6111	D	False
M01368:8:000000000-A3GHV:1:2104:19187:6991	6109	66S2M1D9M1I173M
6111	D	True
M01368:8:000000000-A3GHV:1:1106:13629:12848	6800	3S7M1D1M1D240M
6809	D	True
M01368:8:000000000-A3GHV:1:2109:11237:25755	7603	12S2M1I1M2D235M
7606	D	True
M01368:8:000000000-A3GHV:1:1102:27213:21415	9931	242M1I3M2D2M3S
M01368:8:000000000-A3GHV:1:1105:20381:18275	10517	32S6M1D4M1D2M1D207M
M01368:8:000000000-A3GHV:1:2102:23427:17782	11127	222M3I6M2I5M13S
M01368:8:000000000-A3GHV:1:2111:6146:18232	13388	218M2I9M1I1M2D11M9S
13605	I	True
M01368:8:000000000-A3GHV:1:1114:11462:20755	14640	241M2I3M2D5M
14885	D	True
M01368:8:000000000-A3GHV:1:1112:11649:18276	16365	205M39I4M2I1M
M01368:8:000000000-A3GHV:1:1108:18888:4511	16390	179M62I2M2D2M6S
